Atomia.Store.AspNetMvc/Controllers/CheckoutController.cs

```diff
@@ -33,8 +36,13 @@ public ActionResult Index()
             cartPricingService.CalculatePricing(cart);
 
             // If VAT number was submitted, indicate a VAT check should be made
-            ViewBag.CheckVAT = !String.IsNullOrEmpty(vatDataProvider.VatNumber);
-            
+            ViewBag.CheckVAT = !string.IsNullOrEmpty(vatDataProvider.VatNumber);
+            if (RecaptchaHelper.IsRecaptchaEnabled())
+            {
+                ViewBag.RecaptchaEnabled = true;
+                ViewBag.RecaptchaSiteKey = ConfigurationManager.AppSettings["RecaptchaSiteKey"];

```

Comment:
You are already using the configuration in the RecaptchaHelper. Let's encapsulate this too in it then as well.
```suggestion
                ViewBag.RecaptchaSiteKey = RecaptchaHelper.GetSiteKey();

```